app/javascript/elm/src/SessionsList.elm

```diff
@@ -0,0 +1,188 @@
+port module SessionsList exposing (Model, Msg(..), Session, ShortType, checkedSession, init, main, subscriptions, update, updateSessions, view, viewSession, viewShortType, loadMoreSessions)
+
+import Browser
+import Html exposing (Html, button, dd, div, dl, dt, h1, img, input, label, li, section, span, text, ul)
+import Html.Attributes exposing (checked, class, for, id, src, style, type_)
+import Html.Events exposing (onClick)
+import String exposing (fromInt)
+
+
+---- MODEL ----
+
+
+type alias ShortType =
+    { name : String
+    , type_ : String
+    }
+
+
+type alias Session =
+    { title : String
+    , id : Int
+    , timeframe : String
+    , username : String
+    , shortTypes : List ShortType
+    , selected : Bool -- THIS IS USED IN JS, IN ELM WE TRACK SELECTION IN selectedSessionId. REMOVE WHEN MOVING FETCHING FROM JS TO ELM
+    }
+
+
+type alias Model =
+    { sessions : List Session
+    , selectedSessionId : Maybe Int
+    }
+
+
+init : List Session -> ( Model, Cmd Msg )
+init flags =
+    ( { sessions = flags, selectedSessionId = Nothing }, Cmd.none )
```

Comment:
Does this mean that sending a link with preselected session doesn't work? We should add that before deploying to production.

Reply:
Nice comment. I had to check to be sure but it works because the requests for the sessions is fired after the init. The selected one and the new sessions come in through the port